src/Installers/Windows/AspNetCoreModule-Setup/IIS-Setup/include.wxi

```diff
@@ -2,13 +2,7 @@
 
 <Include>
   <?define DiscoverabilityKeyRoot = "SOFTWARE\Microsoft\IIS Extensions"?>
-  <?define OOBBuildVersion = $(var.BLDVERMAJOR).$(var.BLDVERMINOR).$(var.BLDNUMMAJOR).$(var.BLDNUMMINOR)?>
-
-  <!-- Allow build number as argument or use default -->
-  <?if $(var.BLDNUMMAJOR) = "" ?>
-    <?error Build number undefined ?>
-  <?endif?>
-
```

Comment:
Why remove this ❔

Reply:
Its not set in the new hostoptions usage, so it causes the build to fail, as far as I can tell OOBBuildVersion wasn't being used